tests/tests/test_api_internal.py

```diff
@@ -13,62 +13,72 @@
-        assert 'migration_info' in clean_db['inventory-foobar'].collection_names()
+        assert "inventory-foobar" in clean_db.database_names()
+        assert "migration_info" in clean_db["inventory-foobar"].collection_names()
 
     def test_create_twice(self, internal_client, clean_db):
 
-        _, r = internal_client.create_tenant('foobar')
+        _, r = internal_client.create_tenant("foobar")
         assert r.status_code == 201
 
         # creating once more should not fail
-        _, r = internal_client.create_tenant('foobar')
+        _, r = internal_client.create_tenant("foobar")
         assert r.status_code == 201
 
-
     def test_create_empty(self, internal_client):
         try:
-            _, r = internal_client.create_tenant('')
+            _, r = internal_client.create_tenant("")
         except bravado.exception.HTTPError as e:
             assert e.response.status_code == 400
 
 
 class TestInternalApiDeviceCreate:
-    def test_create_ok(self, internal_client, management_client, clean_db, inventory_attributes):
-        devid = "".join([ format(i, "02x") for i in os.urandom(128)])
-        _, r = internal_client.create_device(device_id = devid,
-                                             attributes=inventory_attributes)
+    def test_create_ok(
+        self, internal_client, management_client, clean_db, inventory_attributes,
+    ):
+        devid = "".join([format(i, "02x") for i in os.urandom(128)])
+        _, r = internal_client.create_device(
+            device_id=devid, attributes=inventory_attributes
+        )
         assert r.status_code == 201
 
-        r, _ = management_client.client.devices.get_devices_id(id=devid,
-                                                               Authorization="foo").result()
+        r, _ = management_client.client.devices.get_devices_id(
+            id=devid, Authorization="foo"
+        ).result()
 
-        self._verify_inventory(inventory_attributes,
-                              [internal_client.Attribute(name=attr.name,
-                                                         value=attr.value,
-                                                         description=attr.description) for attr in r.attributes])
+        self._verify_inventory(inventory_attributes, r.attributes)
 
-    def test_create_twice_ok(self, internal_client, management_client, clean_db, inventory_attributes):
+    def test_create_twice_ok(
+        self, internal_client, management_client, clean_db, inventory_attributes,
+    ):
         # insert first device
-        devid = "".join([ format(i, "02x") for i in os.urandom(128)])
-        _, r = internal_client.create_device(device_id = devid,
-                                             attributes=inventory_attributes)
+        devid = "".join([format(i, "02x") for i in os.urandom(128)])
+        _, r = internal_client.create_device(
+            device_id=devid, attributes=inventory_attributes
+        )
```

Comment:
is this black-formatted?

Reply:
yeah, I have a hook that runs when saving the file. We should probably do a separate PR formatting all the acceptance test sources so we don't get these irrelevant diffs.